src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -293,6 +294,38 @@ internal static string FindBestVersionMatch(VersionRange versionRange, IEnumerab
             return matchingVersion?.ToString();
         }
 
+        private NuGetVersion ResolvePlatformReleaseChannelVersion(IEnumerable<NuGetVersion> bundleVersions, NuGetVersion currentMatchingVersion)
+        {
+            bool isStandardOrExtendedChannel = string.Equals(_platformReleaseChannel, ScriptConstants.StandardPlatformChannelName, StringComparison.Ordinal) ||
+                                   string.Equals(_platformReleaseChannel, ScriptConstants.ExtendedPlatformChannelName, StringComparison.Ordinal);
+
+            bool isLatestOrEmptyChannel = string.Equals(_platformReleaseChannel, string.Empty, StringComparison.Ordinal) ||
```

Comment:
Did you consider inlining this code? or moving closer to usage (with slight adjustments to if else)? That way we will execute this string comparison code only when the first one did return anything.